### ISSUE TITLE
Update colours in ETP shield icon

### ISSUE THREAD

**Issue by AmyYLee** [mozilla-mobile/firefox-ios#6711]
The colours in the ETP shield icon fails contrast test in Dark theme. For consistency we should use the same colouring for Light/Dark theme shield icon on iOS and Fenix. Please see below for colour reference and gradient direction

<img width="749" alt="Screen Shot 2020-05-28 at 17 39 14" src="https://user-images.githubusercontent.com/5992746/83196742-6685a180-a10a-11ea-8465-f331d00bf8f3.png">

**Comment by joeyg** [comment]
@AmyYLee It looks like the shield icon colors come in the image. Would you be able to provide a new asset? https://github.com/mozilla-mobile/firefox-ios/blob/master/Client/Assets/Images.xcassets/tracking-protection-active-block.imageset/trackingprotection-medium.pdf

**Comment by SimonBasca** [comment]
Verified fixed on main 89b961b1ce8 on iPhone XS max (13.4), sim iPhone 6 (12.2), sim iPad Pro 4th gen (13.x).
